tests/main/views/test_frameworks.py

```diff
@@ -3,12 +3,29 @@
 from flask import json
 from freezegun import freeze_time
 import mock
+import pytest
 from sqlalchemy.exc import IntegrityError
 
 from tests.bases import BaseApplicationTest, JSONUpdateTestMixin
 from app.models import db, Framework, SupplierFramework, DraftService, User, FrameworkLot
+from app.models.main import ContactInformation, Supplier
 from tests.helpers import FixtureMixin
 
+from app.constants import (
```

Comment:
same here, is this still `constants`, or `supplier_constants` now?

Reply:
Also, why do we have to import these keys if we define them at class level anyway? Am I missing something? :)

Reply:
We don't need to import them explicitly, however if we just `from app.supplier_constants` then we'd need to reference them as `supplier_constants.KEY_X`, which is a bit verbose.

Reply:
I think you are right 👍